Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -350,37 +350,20 @@ bool CSystem::InitFileSystem_LoadEngineFolders(const SSystemInitParams&)
 }
 
 //////////////////////////////////////////////////////////////////////////
-bool CSystem::InitAudioSystem(const SSystemInitParams& initParams)
+bool CSystem::InitAudioSystem()
 {
-    if (!Audio::Gem::AudioSystemGemRequestBus::HasHandlers())
+    if (!Audio::Gem::SystemRequestBus::HasHandlers())
     {
-        // AudioSystem Gem has not been enabled for this project.
+        // AudioSystem Gem has not been enabled for this project/configuration (e.g. Server).
```

Comment:
A Simpler approach is instead of using the CSystem::InitAudioSystem function.

The AudioSystemGemSystemComponent can in it's `Activate` function register for the "CriticalAssetsCompiled" lifecycle event and then just initialize the Audio system there.